src/extension.ts

```diff
@@ -68,106 +66,112 @@ function activate(context) {
 }
 
 // this method is called when your extension is deactivated
-function deactivate() {
+function deactivate(): boolean {
     return true;
 }
 
 // file signature
-const checkSignature = ( signature ) => {
+function checkSignature( signature: string ): string {
```

Comment:
One thing to note is that I changed `const [functionName] = () => {` (arrow functions) to `function [functionName]`.  The reason I did this is because adding type annotations to functions that you are declaring is a PITA and adds a lot of line length.  I'll reference what I mean below...